webapp/src/snippetBuilder.tsx

```diff
@@ -328,8 +329,9 @@ export function initializeSnippetExtensions(ns: string, extraBlocks: (toolbox.Bl
     const snippetExtensions = getSnippetExtensions();
 
     snippetExtensions.forEach(config => {
-        config.snippetBuilders.forEach(snippet => {
-            if (ns == snippet.namespace) {
+        config.snippetBuilders
```

Comment:
can snippetBuilders be undefined?

Reply:
getSnippetExtensions filters out anything that doesn't include a snippetBuilders property so it will always be there.